account/manager.go

```diff
@@ -107,6 +108,65 @@ func (wt witnessType) witnessSize() (int, error) {
 	}
 }
 
+// Action is a type of account modification.
+type Action string
+
+const (
+	CREATE   Action = "create"
+	DEPOSIT  Action = "deposit"
+	WITHDRAW Action = "withdraw"
+	RENEW    Action = "renew"
+	CLOSE    Action = "close"
+)
+
+type TxLabel struct {
+	Account AccountTxLabel `json:"account"`
+}
+
+type AccountTxLabel struct {
+	Key           string         `json:"key"`
+	Action        Action         `json:"action"`
+	ExpiryHeight  uint32         `json:"expiry_height"`
+	OutputIndex   uint32         `json:"output_index"`
+	IsExpirySpend bool           `json:"expiry_spend"`
+	TxFee         btcutil.Amount `json:"tx_fee"`
+	BalanceDiff   btcutil.Amount `json:"balance_diff"`
+}
+
+// actionTxLabel returns a transaction label for use with account
+// modification actions.
+func actionTxLabel(account Account, action Action, isExpirySpend bool,
+	txFee btcutil.Amount, balanceDiff btcutil.Amount,
+) (*string, error) {
+
+	prefixTag := "poold -- "
+
+	acctKey := account.TraderKey.PubKey.SerializeCompressed()
+	label := AccountTxLabel{
+		Key:           string(acctKey),
```

Comment:
The key should be hex formatted, otherwise this will directly convert the byte slice into ASCII characters.